packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -262,6 +295,41 @@ class UserSessionManager implements IUserSessionManager {
     return this.storage?.get(userSessionStorageKeys.initialReferringDomain) || null;
   }
 
+  /**
+   * Fetches session tracking information from storage
+   * @returns
+   */
+  getSession(): Nullable<SessionInfo> {
```

Comment:
please rename to getSessionFromStorage to avoid confusion with session in state

Reply:
Addressed